packages/decentraland-ecs/src/decentraland/Scene.ts

```diff
@@ -0,0 +1,57 @@
+import { onEnterSceneObservable, onLeaveSceneObservable } from './Events'
+
+/**
+ * @public
+ */
+export namespace Scene {
+  /**
+   * @public
+   */
+  export type OnEnterLeaveOptions = {
+    onlyCurrentUser: boolean
+  }
+
+  /**
+   * TThese events are triggered after a character enters the scene.
+   * @public
+   */
+  export function OnEnterScene(callback: (event: IEvents['onEnterScene']) => void, options?: OnEnterLeaveOptions) {
```

Comment:
1. For consistency sake, this should be an observable also
2. Functions are camelCase not PascalCase